src/einsteinpy/symbolic/tensor.py

```diff
@@ -31,8 +32,29 @@ def __init__(self, arr):
             raise TypeError("Only multi-dimensional list or Sympy Array is expected")
 
     def __getitem__(self, index):
+
         return self.arr[index]
 
+    def __str__(self) -> str:
+        """
+        Returns a String with a readable representation of the object of class Tensor
+
+        """
+        str_with_the_result = "Tensor Object: "
+        for tensor in self.arr:
+            str_with_the_result += "\n  * {}".format(tensor)
+        return str_with_the_result
```

Comment:
I would say the best way to create `__str__` is use the `__str__` function of `self.arr` itself.
Something like:
```
s = "Tensor\n"
s += self.arr.__str__()
return s
```